tedium.ts

```diff
@@ -386,11 +386,36 @@ function connectToGithub() {
  * Returns a promise of the hydrolysis.Analyzer with all of the info loaded.
  */
 async function analyzeRepos() {
-  const dirs = fs.readdirSync('repos/');
+  const dirsToConsider: string[] = [];
   const htmlFiles: string[] = [];
 
-  for (const dir of dirs) {
+  for (const dir of fs.readdirSync('repos/')) {
+    dirsToConsider.push(path.join('repos', dir));
     for (const fn of fs.readdirSync(path.join('repos', dir))) {
+      try {
+        const stat = await promisify(fs.stat)(path.join('repos', dir, fn));
```

Comment:
Other work can continue during the async call - I believe the VM is blocked during the sync call